electron/ui/src/views/Optimization/Optimization.js

```diff
@@ -264,49 +264,20 @@ export default function Optimization(props) {
           </Grid>
           <Grid item xs={columnWidths[1]} style={styles.gridItems}>
           </Grid>
-
           <Grid item xs={columnWidths[0]} style={styles.gridItems}>
           <Collapse in={showAdvancedOptions} timeout="auto" unmountOnExit>
-            <Box sx={styles.settingName}>
-              <p>Scale Model</p>
-              <Tooltip title={descriptions.scaleModel} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
-            <Box sx={styles.settingName}>
-              <p>Pipeline Capacity</p>
-              <Tooltip title={descriptions.pipelineCapacity} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
-            <Box sx={styles.settingName}>
-              <p>Pipeline Cost</p>
-              <Tooltip title={descriptions.pipelineCost} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
-            <Box sx={styles.settingName}>
-              <p>Node Capacity</p>
-              <Tooltip title={descriptions.nodeCapacity} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
-            <Box sx={styles.settingName}>
-              <p>Infrastructure Timing</p>
-              <Tooltip title={descriptions.infrastructureTiming} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
-            <Box sx={styles.settingName}>
-              <p>Subsurface Risk</p>
-              <Tooltip title={descriptions.subsurfaceRisk} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
-            <Box sx={styles.settingName}>
-              <p>Removal Efficiency Method</p>
-              <Tooltip title={descriptions.removalEfficiencyMethod} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
-            <Box sx={styles.settingName}>
-              <p>Desalination Model</p>
-              <Tooltip title={descriptions.desalinationModel} placement="right-start"><IconButton><InfoIcon fontSize='small'/></IconButton></Tooltip>
-            </Box>
```

Comment:
no they're all still there. just rewrote this as a loop ([lines 269-276](https://github.com/project-pareto/pareto-ui/blob/e53062bcd598f33ce5ada3a9a4c6fb28f1868667/electron/ui/src/views/Optimization/Optimization.js#L269-L276) in the updated file) rather than repeating the same components for each option